eos/maths/gegenbauer-polynomial.hh

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2023 Viktor Kuschke
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#ifndef EOS_GUARD_EOS_MATHS_GEGENBAUER_POLYNOMIALS_HH
+#define EOS_GUARD_EOS_MATHS_GEGENBAUER_POLYNOMIALS_HH 1
+
+#include <eos/maths/power-of.hh>
+
+#include <vector>
+
+namespace eos
+{
+    class GegenbauerPolynomial
+    {
+        private:
+            const unsigned int order; // order of the Gegenbauer polynomial
```

Comment:
Coding style: The names of private member variables should be prefixed with an underscore, i.e., ``_order``.